src/store/timetravel.js

```diff
@@ -65,6 +61,15 @@ export default {
     const originalReplaceState = store.replaceState;
     store.replaceState = function overrideReplaceState(...args) {
       if (window.api) { window.api.config.onChange(); }
+      const newState = args[0];
+
+      // non timetravel props
+      // TODO: ask brian about config syntax for this
+      newState.application.scale = this.state.application.scale;
+      newState.project.view = this.state.project.view;
+      newState.project.grid = this.state.project.grid;
+      newState.project.previous_story = this.state.project.previous_story;
+
```

Comment:
This ignores properties on the state which we do not want affected by timetravel